openapi/paths/getters/getAsset.json

```diff
@@ -17,6 +17,22 @@
           "type": "integer",
           "format": "int64"
         }
+      },
+      {
+        "name": "heightStart",
+        "in": "query",
+        "description": "Define the start height for trading information over a given period - if not given the last 24h are taken as trading period",
+        "schema": {
+          "type": "integer"
+        }
+      },
+      {
+        "name": "heightEnd",
+        "in": "query",
+        "description": "Define the end height for trading information over a given period - if not given the 24h are taken as trading period",
```

Comment:
"Define the end height for trading information over a given period - if not given the current block is taken as end height"